web/src/main/java/no/nav/modiapersonoversikt/legacy/sak/service/saf/graphql/SafGraphqlServiceImpl.kt

```diff
@@ -123,9 +128,33 @@ class SafGraphqlServiceImpl : SafService {
         baksystem = baksystem.plus(Baksystem.SAF)
         temakode = journalpost.tema?.name
         temakodeVisning = journalpost.temanavn
+        kanalNavn = journalpost.kanalnavn
+        kanalType = hentKanalType(journalpost)
+        antallRetur = journalpost.antallRetur
         return this
     }
 
+    private fun hentKanalType(journalpost: HentBrukersDokumenter.Journalpost): KanalType =
+        when {
```

Comment:
Kan vi bruke enum verdiene her? Slik som det gjøres for `getDokumentStatus`, siden da får vi litt typesikkerhet også. :)

Reply:
ok